### PR TITLE
Update nl.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

**Review by BellezaEmporium — DISMISSED** [review]

**Review by Sphinxroot — DISMISSED** [review]

File: streams/nl.m3u [hunk 1]
```diff
@@ -119,8 +121,10 @@ https://ms7.mx-cd.net/dtv-10/91-475821/Regio8TV.smil/playlist.m3u8
 https://ms2.mx-cd.net/dtv-02/260-2403096/Regio90TV.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="RegioTVNieuws.nl" status="online",Regio TV Nieuws (1080p) [Not 24/7]
 http://highvolume04.streampartner.nl/regiomedia/regiomedia/playlist.m3u8
-#EXTINF:-1 tvg-id="RN7.nl" status="blocked",Radio RN7 (576p) [Geo-blocked]
+#EXTINF:-1 tvg-id="" status="blocked",Radio RN7 (576p) [Geo-blocked]
 https://streaming.rn7.nl/rn7live_abr/visualradio/playlist.m3u8
+#EXTINF:-1 tvg-id="RN7.nl",RN7 TV (1080p)
+https://streaming.rn7.nl/rn7live_abr/tv/rn7tv/live_1080/playlist.m3u8
```
Comment (Alstruit) [inline streams/nl.m3u:127]:
```suggestion
#EXTINF:-1 tvg-id="RN7.nl",RN7 TV (1080p)
#EXTVLCOPT:http-referrer=https://player.rn7.nl/
https://streaming.rn7.nl/rn7live_abr/visualradio/playlist.m3u8
```
Requires Header

**Review by Alstruit — DISMISSED** [review]

File: streams/nl.m3u [hunk 2]
```diff
@@ -59,8 +59,10 @@ https://streamingserver02.omroepnuenen.nl/lon/lonweb_720p/playlist.m3u8
 https://593aed234297b.streamlock.net/meervandaag/meervandaag/playlist.m3u8
 #EXTINF:-1 tvg-id="ML5TV.nl" status="error",ML5 TV (480p)
 https://ms2.mx-cd.net/dtv-02/204-1147034/3ML_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ML5TV.nl",ML5 TV (480p)
```
Comment (BellezaEmporium) [inline streams/nl.m3u:62]: Same stream than the one above, different server.

File: streams/nl.m3u [hunk 3]
```diff
@@ -203,51 +210,189 @@ https://ms7.mx-cd.net/tv/99-660295/WOS.smil/playlist.m3u8
...
-#EXTINF:-1 tvg-id="360TuneBox.nl",360TuneBox
+#EXTINF:-1 tvg-id="360TuneBox.nl",360TuneBox (576p)
 https://vod1.laki.eu/360tune/playlist.m3u8
-#EXTINF:-1 tvg-id="FashionBox.nl",FashionBox
-https://vod1.laki.eu/fashionbox/video.m3u8
-#EXTINF:-1 tvg-id="FightBox.nl",FightBox
-https://vod1.laki.eu/fightbox/video.m3u8
-#EXTINF:-1 tvg-id="FastFunBox.nl",Fast&FunBox
+#EXTINF:-1 tvg-id="FashionBox.nl",FashionBox (576p)
+https://vod1.laki.eu/fashionbox/video.m3u8
```
Comment (BellezaEmporium) [inline streams/nl.m3u:256]:
```suggestion
https://vod1.laki.eu/fashionbox/video.m3u8
```

File: streams/nl.m3u [hunk 4]
```diff
@@ -203,51 +210,189 @@ https://ms7.mx-cd.net/tv/99-660295/WOS.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="360TuneBox.nl",360TuneBox (576p)
 https://vod1.laki.eu/360tune/playlist.m3u8
-#EXTINF:-1 tvg-id="FashionBox.nl",FashionBox
+#EXTINF:-1 tvg-id="FashionBox.nl",FashionBox (576p)
 https://vod1.laki.eu/fashionbox/video.m3u8
-#EXTINF:-1 tvg-id="FightBox.nl",FightBox
-https://vod1.laki.eu/fightbox/video.m3u8
-#EXTINF:-1 tvg-id="FastFunBox.nl",Fast&FunBox
+#EXTINF:-1 tvg-id="FightBox.nl",FightBox (576p)
+https://vod1.laki.eu/fightbox/video.m3u8
```
Comment (BellezaEmporium) [inline streams/nl.m3u:258]:
```suggestion
https://vod1.laki.eu/fightbox/video.m3u8
```

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**